service/switchboardapp/src/test/java/com/switchboard/app/fixture/DecoderFixture.java

```diff
@@ -0,0 +1,26 @@
+package com.switchboard.app.fixture;
+
+import com.switchboard.app.domain.DecoderEntity;
+import com.switchboard.app.domain.DeviceEntity;
```

Comment:
`import com.switchboard.app.domain.DecoderEntity;` needs to change to `import com.switchboard.app.entity.DecoderEntity;`

`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated.